addons/marker_flags/CfgWeapons.hpp

```diff
@@ -0,0 +1,55 @@
+class CfgWeapons {
+    class ACE_ItemCore;
+    class CBA_MiscItem_ItemInfo;
+
+    class GVAR(white): ACE_ItemCore {
+        GVAR(vehicle) = GVAR(white);
+        author = ECSTRING(common,ACETeam);
+        scope = 2;
+        displayName = CSTRING(white);
+        model = "\a3\Structures_F_Orange\Signs\Special\FlagMarker_F.p3d";
```

Comment:
```suggestion
        // model = "\a3\Structures_F_Orange\Signs\Special\FlagMarker_F.p3d";
```
Maybe the dlc warnings will be too confusing for people, it might be best to leave it out for now

Reply:
What would we see when dropped on the ground then?

Reply:
The default box item probably.

Reply:
@BrettMayson
Made this model for the cfgWeapon in blender (lower right, the blag flag is vanilla cfgVeh model whille we don't need to modify)
just the pole no flag, but I think that's fine as before all models used the same white flag.

let me know if this ok to push

![20220506151333_1](https://user-images.githubusercontent.com/9376747/167210412-9037d476-23dc-4715-8eda-c0d5fa4104e5.jpg)

Reply:
What about a DLC usability check around it ?

Reply:
I think the pole with no flag is great for a model